docs/data-sources/organization_members.md

```diff
@@ -0,0 +1,23 @@
+---
+page_tile: "Splunk Observability Cloud - signalfx_organization_members
+description: |-
+    Allows for members to be queried and used as part of other resources. Requires the supplied token to have Admin priviledges.
+---
+
+# Data Source: signalfx_organization_members
+
+Allows for members to be queried and used as part of other resources. Requires the supplied token to have Admin priviledges.
+
+<!-- schema generated by tfplugindocs -->
+## Schema
+
+### Required
+
+- `emails` (List of String) A list of email address that can be matched against existing users of the organization.
+
+### Read-Only
+
+- `id` (String) The ID of this resource.
+- `users` (List of String) Provides a list of user ids that match the emails provided returned in order of emails provided.
+
```

Comment:
Same:

- `id`. String. The ID of this resource.
- `users`. List of strings. Provides a list of user ids that match the emails provided returned in order of emails provided.